### ISSUE TITLE
Request to add Openspeedtest

### ISSUE THREAD

**Issue by YoshiKill** [sirpdboy/netspeedtest#52]
Hello, thank you for great speed package. I have one request. There is one nice project for speed Analise [OpenSpeedTest™](https://openspeedtest.com/?Run&ref=Github)  can you add it in to this packet of utility? It will be greet.

**Comment by sirpdboy** [comment]
yes  add [OpenSpeedTest™](https://openspeedtest.com/?Run&ref=Github)
